lib/rack/tracker/google_analytics/google_analytics.rb

```diff
@@ -28,6 +31,18 @@ def tracker
     options[:tracker].respond_to?(:call) ? options[:tracker].call(env) : options[:tracker]
   end
 
+  def tracker_options
+    @tracker_options ||= begin
+      tracker_options = {}
```

Comment:
i know its a matter of taste, but personally i don't like these temporary variables, `tap` is your fried to get rid of 'em